docs/oxAuth_swagger.yaml

```diff
@@ -0,0 +1,2030 @@
+          description: Hint to the Authorization Server about the login identifier the End-User might use to log in (if necessary). 
+          schema:
+            type : string
+        - name: acr_values
+          in: query
+          required: false
+          description: Requested Authentication Context Class Reference values. Space-separated string that specifies the acr values that the Authorization Server is being requested to use for processing this Authentication Request, with the values appearing in order of preference.
+          schema:
+            type : string
+        - name: amr_values
+          in: query
+          required: false
+          description: AMR Values.
+          schema:
+            type : string
+        - name: request
+          in: query
+          required: false
+          description: This parameter enables OpenID Connect requests to be passed in a single, self-contained parameter and to be optionally signed and/or encrypted. The parameter value is a Request Object value. It represents the request as a JWT whose Claims are the request parameters.
+          schema:
+            type : string
+        - name: request_uri
+          in: query
+          required: false
+          description: This parameter enables OpenID Connect requests to be passed by reference, rather than by value. The request_uri value is a URL using the https scheme referencing a resource containing a Request Object value, which is a JWT containing the request parameters.
+          schema:
+            type : string
+        - name: request_session_id
+          in: query
+          required: false
+          description: Request session id.
+          schema:
+            type : string
+        - name: session_id
+          in: query
+          required: false
+          description: Session id of this call.
+          schema:
+            type : string
+        - name: origin_headers
+          in: query
+          required: false
+          description: Origin headers. Used in custom workflows.
+          schema:
+            type : string
+        - name: code_challenge
+          in: query
+          required: false
+          description: PKCE code challenge.
+          schema:
+            type : string
+        - name: code_challenge_method
+          in: query
+          required: false
+          description: PKCE code challenge method.
+          schema:
+            type : string
+        - name: custom_response_headers
+          in: query
+          required: false
+          description: Custom Response Headers.
+          schema:
+            type : string
+        - name: claims
+          in: query
+          required: false
+          description: Requested Claims.
+          schema:
+            type : string
+        - name: auth_req_id
+          in: query
+          required: false
+          description: CIBA authentication request Id.
+          schema:
+            type : string
+      responses:
```

Comment:
the main response is 302, where is it ?

Reply:
I went through the code in AuthorizeRestWebServiceImpl.java and did not find 302 hence did not mention it.
302 is specified in @ApiResponses in interface AuthorizeRestWebService and thought its not in sync with the actual implementation.
Please let me know if I need correction

Reply:
![Authorize_Error_Response](https://user-images.githubusercontent.com/43700552/75666457-d74c2780-5c9b-11ea-8a69-e4d63e2dbb88.jpg)